src/extensions/default/BrambleUrlCodeHints/style.css

```diff
@@ -106,3 +106,956 @@
 #selfie-video-bg.on {
     opacity: 1;
 }
+
+#filter-list {
```

Comment:
Thanks for the feedback @Pomax @humphd !
It's a shame we can't use the CSS solution with the canvas.
I went through the documentation for each of the links and I think that something like caman.js is the most promising. 
I'll try to get something prototyped within the next few days.